src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -119,7 +124,15 @@ Env::close(NetClock::time_point closeTime,
     // Round up to next distinguishable value
     closeTime += closed()->info().closeTimeResolution - 1s;
     bundle_.timeKeeper->set(closeTime);
-    app().getOPs().acceptLedger(consensusDelay);
+    // Go through the rpc interface unless we need to simulate
+    // a specific consensus delay.
+    if (consensusDelay)
+        app().getOPs().acceptLedger(consensusDelay);
```

Comment:
I'm not really fond of this but I won't hold it up. But we should keep our eye on it, the technique seems fragile.